### DISCUSSION TITLE
A little bit of thank you in 2024!

### DISCUSSION THREAD

**Discussion by gAlleb** [savonet/liquidsoap#3629] [General]
Hi, folks! I am in some grateful mood so why not)

I'd like to say `thank you` to the Liquidsoap crew for such a great project. It definitely adds to the everchanging media community. And I hope this year lots of aimed goals will be reached and accomplished! 

I can't help but to give credit to @toots and @smimram! And to all collaborators and authors (@vitoyucepi, @busterneece, @moonbase59 and more... Thanks to Mr. @mikulskii who blogs about liquidsoap in Russian community) and other great people who commit every day to this incredible peace of software.

And I I can't help but to mention the patience and restraint with which they answer questions, sometimes quite stupid ones :) (especially @vitoyucepi, who has the super ability to help,  thank you:).

@toots, @smimram I wish you to achieve new goals this year and further develop this amazing project: streaming and clock model, concurrency (OCaml 5), JS and more and more. I sincerely wish you to live and prosper!

Thanks! Everyday I can learn something new and I thank y'all for that. And wish `Liquidsoap` someday to became a standard! (Well, it surely looks to me like that) :)

Thank you!

**Comment by toots** [comment]
Thank you! We're really happy with the community that's been gathering around the project and really appreciate y'all enthusiasm and contributions!
